statsmodels/tsa/arima_model.py

```diff
@@ -973,6 +973,10 @@ def fit(self, start_params=None, trend='c', method="css-mle",
         armafit.mle_settings = mlefit.mle_settings
         return ARMAResultsWrapper(armafit)
 
+    # base class of "from_formula" is "class Model(object)"
+    @classmethod
+    def from_formula(cls, formula, data, subset=None, drop_cols=None, *args, **kwargs):
+            raise NotImplementedError("from_formula is not supported for ARMA models.")
```

Comment:
are these below 80 characters per line

Reply:
I'm not aware of the 80 characters limitation, the prototype of the method is 83 character (as the original) and the second line has 75 characters (87 if I count white-space )

Reply:
I have read about the 80 pep8, I can refactor the error line to be <80, but should I also refactor the line of the method's prototype that is too loog (if yes, the original prototype, the derived or both)?

Reply:
fix just where you change things or your additions
It's better not to add style changes for code not directly related to chanes in order to keep the noise out of PRs.
(We still need some style changes in many modules, but those are better left to separate PRs, and when they don't create too many merge conflicts with existing PRs.)